idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/utils/IdeUtils.kt

```diff
@@ -52,6 +53,7 @@ import org.jetbrains.kotlin.utils.addToStdlib.safeAs
 
 internal fun <A> isNotNull(a: A?): Boolean = a?.let { true } ?: false
 
+internal fun KtTypeReference.getType(): KotlinType? = analyze()[BindingContext.TYPE, this]
```

Comment:
this shouldn't be part of the Utils, when it is specific to a plugin feature. Where is this used?

Reply:
only for foldings.. moving it to foldingUtils for now 👍